torch/_inductor/select_algorithm.py

```diff
@@ -1070,9 +1070,10 @@ def no_op(*args, **kwargs):
             ):
                 return no_op
 
-            # TODO - debug issue
-            if torch.version.hip:
-                return no_op
+            # TODO(eelison) - debug issue
```

Comment:
cc @eellison

Reply:
Nice - ran into test failure at one point with sdpa for this. there shouldn't be an issue so if tests pass great.

Reply:
Thanks! If there is no issue I can clean it up